src/go/rdctl/cmd/snapshotCreate.go

```diff
@@ -38,7 +44,17 @@ func createSnapshot(args []string) error {
 		return nil
 	}
 
-	if _, err := manager.Create(name, snapshotDescription); err != nil {
+	// Ideally we would not use the deprecated syscall package,
+	// but it works well with all expected scenarios and allows us
+	// to avoid platform-specific signal handling code.
```

Comment:
Please see comments — we should file a new issue to have a platform-specific code path for Windows because that is annoying :p

Reply:
Created https://github.com/rancher-sandbox/rancher-desktop/issues/6049 for this.